src/test/java/io/lettuce/core/cluster/RedisClusterPasswordSecuredSslTest.java

```diff
@@ -47,7 +47,7 @@
     public static final String SLOT_1_KEY = "8HMdi";
     public static final String SLOT_16352_KEY = "UyAa4KqoWgPGKa";
 
-    public static RedisURI redisURI = RedisURI.builder().redis(host(), CLUSTER_PORT_SSL_1).withPassword("foobared")
+    public static RedisURI redisURI = RedisURI.Builder.redis(host(), CLUSTER_PORT_SSL_1).withPassword("foobared")
```

Comment:
Are these changes required?

Reply:
They aren't. As I dug around I noticed a few intellij inspection warnings that seemed easy to clean up. I'll back them out.